### PR TITLE
Google safe browsing bug

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27321

## Description
When there are no result , dbot score now returns 0 to context

## Screenshots
image

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
> looks good, though the print screen attach shows `phishtank` as the vendor, not `google safe browsing`.
> please add a task for the test playbook testing that as well.

It was by mistake in the screen shot, changed it. In the code its google safe browsing

File: Packs/GoogleSafeBrowsing/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Google Safe Browsing
+- Fixed an issue when there are no results Dbot score returns 0.
```
Comment (yuvalbenshalom) [inline Packs/GoogleSafeBrowsing/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where the url command should return Dbot score 0 when no results are found.
```

**Comment by bakatzir** [comment]
@kirbles19 a 1 command integration without readme, I guess it was not reviewed by your team. consider if you wish to go over the yml now, or only the rn.

File: Packs/GoogleSafeBrowsing/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Google Safe Browsing
+- Fixed an issue where the url command should return Dbot score 0 when no results are found.
```
Comment (amshamah419) [inline Packs/GoogleSafeBrowsing/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where the `url` command should return a Dbot score of 0 when no results are found.
```

File: Packs/GoogleSafeBrowsing/Integrations/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+| proxy | Use system proxy settings | False |
+| insecure | Trust any certificate \(not secure\) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### url
+***
+Check URL Reputation
```
Comment (amshamah419) [inline Packs/GoogleSafeBrowsing/Integrations/README.md:24]:
```suggestion
Checks the reputation of a URL.
```

File: Packs/GoogleSafeBrowsing/Integrations/README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| URL.Data | unknown | Bad URLs found | 
+| URL.Malicious.Vendor | unknown | For malicious URLs, the vendor that made the decision | 
+| URL.Malicious.Description | unknown | For malicious URLs, the reason for the vendor to make the decision | 
+| DBotScore.Indicator | unknown | The indicator we tested | 
+| DBotScore.Type | unknown | The type of the indicator | 
+| DBotScore.Vendor | unknown | Vendor used to calculate the score | 
+| DBotScore.Score | unknown | The actual score |
```
Comment (amshamah419) [inline Packs/GoogleSafeBrowsing/Integrations/README.md:47]:
```suggestion
| URL.Data | string | Bad URLs found | 
| URL.Malicious.Vendor | string | For malicious URLs, the vendor that made the decision | 
| URL.Malicious.Description | string | For malicious URLs, the reason for the vendor to make the decision | 
| DBotScore.Indicator | string | The indicator we tested | 
| DBotScore.Type | string | The type of indicator | 
| DBotScore.Vendor | string | Vendor used to calculate the score | 
| DBotScore.Score | int | The actual score | 
```

**Pull request merged**